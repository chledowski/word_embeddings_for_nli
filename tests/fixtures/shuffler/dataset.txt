2 ab S2
5 abcde S5
1 a S1
6 abcdef S6
3 abc S3
4 abcd S4