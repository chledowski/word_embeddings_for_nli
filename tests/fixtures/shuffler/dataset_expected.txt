1 a S1
2 ab S2
3 abc S3
4 abcd S4
5 abcde S5
6 abcdef S6